app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) != 3 or parts[0] != "cp":
+        print("Invalid command")
+        return
+
+    source_file = parts[1]
+    dest_file = parts[2]
+
+    if source_file == dest_file:
+        return
```

Comment:
The function should print a message indicating that the operation is not performed when the source and destination files are the same, as per the task description: 'It must do nothing in case the user is trying to copy file to file with the same name.' Consider adding a print statement to inform the user.